CRM/Contact/BAO/Contact.php

```diff
@@ -2730,7 +2730,7 @@ public static function getCountComponent($component, $contactId, $tableName = NU
 
       case 'group':
 
-        return CRM_Contact_BAO_GroupContact::getContactGroup($contactId, "Added", NULL, TRUE);
+        return CRM_Contact_BAO_GroupContact::getContactGroup($contactId, "Added", NULL, TRUE, FALSE, FALSE, TRUE, NULL, TRUE);
```

Comment:
Will this cause cause smart groups to be resolved? if so that might be problematic on sites with a large number of active smart groups

Reply:
@seamuslee001, no it will not. As I wrote in the comments (above):

> The count only includes Smart Groups to which the contact has been _explicitly_ added - not all Smart Groups whose search criteria include the contact. This avoids the [documented](https://issues.civicrm.org/jira/browse/CRM-11068) [problems](https://issues.civicrm.org/jira/browse/CRM-12466) with recalculating all Smart Group memberships, so there is no significant performance hit.

The only thing that this patch does is to change the value of the **$includeSmartGroups** argument passed to the **CRM_Contact_BAO_GroupContact::getContactGroup** function. If you look at the code of that function, you'll see that that argument only has one effect: It adds the clause "**AND saved_search_id IS NULL**" to a query. There is no way that that change could cause Smart Groups to be recalculated.

Reply:
I agree